### ISSUE TITLE
Documentation and/or UI for colors= QS parameter

### ISSUE THREAD

**Issue by jteresco** [TravelMapping/Web#610]
The colors= QS parameter has been restored in the current mapview and showroute implementations, and expanded as discussed on the forum and in other issues.

This functionality should be documented in the manual, and/or a user interface to set these custom colors should be implemented.

**Comment by jteresco** [comment]
Idea for selecting colors: have the color squares in the Mapview legend bring up a color picker.

**Comment by michihdeu** [comment]
yep, clicking on the color squares to open an editor! Then, an option whether it is only for this system, tier-dependent or whatever (with the "always update" option the systems on mapview can change with every mouse move!).

The colors should be stored in cookies (as mentioned by mapcat on the forum) in case one changes e.g. from mapview to showroute etc.

On showroute.php we might have an icon next to "View connected route" or "Jump to similar route" to open the color editor.
